recipes-kernel/linux/linux-imx_5.10.bb

```diff
@@ -13,20 +13,20 @@ i.MX Family Reference Boards. It includes support for many IPs such as GPU, VPU
 require recipes-kernel/linux/linux-imx.inc
 
 LICENSE = "GPLv2"
-LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
 DEPENDS += "lzop-native bc-native"
 
-SRCBRANCH = "imx_5.4.70_2.3.0"
-LOCALVERSION = "-2.3.2"
-SRCREV = "5f2220e87dd8a8fb86f524ea7945ab1d4155bc73"
+SRCBRANCH = "lf-5.10.y"
+LOCALVERSION = "-lts-5.10.y"
```

Comment:
The local version is better to point to the GA release it is using.

Reply:
Okay, so to follow the last change, set it to `-2.0.0`?